src/Core/NuGet/Package.nuspec

```diff
@@ -22,17 +22,17 @@
         <dependency id="Microsoft.Diagnostics.Tracing.EventSource.Redist" version="1.1.28"/>
       </group>
       <group targetFramework="net45">
-        <dependency id="System.Diagnostics.DiagnosticSource" version="4.4.0" />
+        <dependency id="System.Diagnostics.DiagnosticSource" version="4.4.0-beta-25130-03" />
```

Comment:
:( was wrong again.
This way Nuget tried to reference the lowest possible version and ended up with the version that did not have Activity...